src/cli/lib/printer/printer.ts

```diff
@@ -47,6 +49,19 @@ type Style = {
 	decoration?: Decoration;
 };
 
+const printLoader = () => {
+	const p = ['⠙', '⠘', '⠰', '⠴', '⠤', '⠦', '⠆', '⠃', '⠋', '⠉'];
+	let x = 0;
+	return setInterval(() => {
+		process.stdout.write('\r' + p[x++]);
+		x &= 3;
+	}, 250);
+};
+
+const removeLoader = () => {
```

Comment:
We don't have a linter rule to force typescript to specify the return value, but just in case you want to practice with it, you can add what the `removeLoader` method returns.

`const removeLoader = (): void => {`